### PR TITLE
Fix quest condition info data and reduce duplicate code

### PR DESCRIPTION
**Pull request by bschultz**

### CODE REVIEW

File: src/models/pokestop.js [hunk 1]
```diff
@@ -289,14 +289,11 @@ class Pokestop {
...
                 case ConditionType.ThrowType:
-                    if (info.with_throw_type.throw_type > 0) {
-                        infoData['throw_type_id'] = info.with_throw_type.throw_type;
-                    }
-                    break;
                 case ConditionType.ThrowTypeInARow:
                     if (info.with_throw_type.throw_type > 0) {
                         infoData['throw_type_id'] = info.with_throw_type.throw_type;
                     }
+                    infoData['hit'] = info.with_throw_type.hit
```
Comment (bschultz) [inline src/models/pokestop.js:296]: @versx Do we maybe need the data for `in_a_row` here too?

Comment (versx) [reply src/models/pokestop.js:296]: Hmm guess I missed that one.

Comment (bschultz) [reply src/models/pokestop.js:296]: I mean the info for `ThrowType` and `ThrowTypeInARow` is currently the same and RDM also didn't have the boolean info for `in_a_row`, but I guess we could just add it?

Comment (bschultz) [reply src/models/pokestop.js]: Oh wait it's already defined by condition type...So not needed 😂 🙈

Comment (versx) [reply src/models/pokestop.js:296]: Oh shoot you're right 😂

**Pull request merged**